### PR TITLE
DOC-6275 added key for v23

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-6275
- continuation of changing the nav and renaming 'Use Cloud Storage' page
- fixes dropdown error for v23.1

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -382,7 +382,6 @@ layout: null
 /docs/cockroachcloud/migrate-from-free-to-dedicated /docs/cockroachcloud/migrate-from-serverless-to-dedicated.html 301
 /docs/cockroachcloud/free-faqs /docs/cockroachcloud/serverless-faqs.html 301
 /docs/v22.2/use-cloud-storage-for-bulk-operations.html /docs/v22.2/use-cloud-storage.html 301
-/docs/v23.1/use-cloud-storage-for-bulk-operations.html /docs/v23.1/use-cloud-storage.html 301
```
Comment (nickvigilante) [inline _redirects:385]: Any reason you're moving the redirect here? I think it's still necessary to include.

**Review by nickvigilante — COMMENTED** [review]
One question on a change.

File: _redirects (hunk 1 above)
Comment (ashleykwtam) [reply _redirects:385]: added back above

**Comment by ashleykwtam** [comment]
thanks Nick!

**Pull request merged**